app/javascript/dashboard/store/modules/customViews.js

```diff
@@ -61,6 +61,13 @@ export const actions = {
       commit(types.SET_CUSTOM_VIEW_UI_FLAG, { isCreating: false });
     }
   },
+  setCustomViewCount: async ({ commit }, { filter }) => {
+    try {
+      commit(types.SET_CUSTOM_VIEW_COUNT, filter);
+    } catch (error) {
+      throw new Error(error);
+    }
```

Comment:
The try catch does not really do anything, let's log it to sentry and let it fail silently